guides/v2.2/cloud/project/new-relic.md

```diff
@@ -149,3 +149,13 @@ You can use the high-level data captured by New Relic APM to see what is happeni
 -  The New Relic worst-case requests are not averages, but edge cases; whereas, Blackfire provides more of the average worst-case requests.
 -  Blackfire is not detailed in determining and displaying queries giving you trouble, but you can use New Relic to find those.
 -  Background processes, like Cron job, can cause issues. These background processes are not monitored by New Relic, but you can watch them using Blackfire.
+
+## Change account owner
+
+See [New Relic's documentation][1].
```

Comment:
Why? Seems early for this.

Reply:
The idea is to immediately direct readers New Relic's docs to learn how to do this. I didn't feel like a long explanation was necessary.